meta-buildpacks/nodejs-function/package.toml

```diff
@@ -5,10 +5,7 @@ uri = "."
 uri = "libcnb:heroku/nodejs-engine"
 
 [[dependencies]]
-uri = "libcnb:heroku/nodejs-npm-engine"
-
-[[dependencies]]
-uri = "libcnb:heroku/nodejs-npm-install"
+uri = "../../buildpacks/npm"
```

Comment:
This location may not exist in the future. Should we use the remote uri on dockerhub instead?

Reply:
Well, the implication of this is we would have to keep the bash-based npm buildpack around a bit longer.